Makefile

```diff
@@ -52,18 +52,20 @@ cross:
 	GOOS=windows $(MAKE) build BUILDTAGS="$(BUILDTAGS) $(BUILD_TAGS_WINDOWS_CROSS)"
 	GOOS=darwin $(MAKE) build BUILDTAGS="$(BUILDTAGS) $(BUILD_TAGS_DARWIN_CROSS)"
 
-tools: tools.timestamp
+tools: .install.gitvalidation .install.golangci-lint
 
-tools.timestamp: Makefile
-	@GO111MODULE="off" go get -u $(BUILDFLAGS) golang.org/x/lint/golint
+.install.gitvalidation:
+	if [ ! -x "$(GOBIN)/git-validation" ]; then \
+		GO111MODULE="off" go get $(BUILDFLAGS) github.com/vbatts/git-validation; \
+	fi
+
+.install.golangci-lint: .gopathok
 	if [ ! -x "$(GOBIN)/golangci-lint" ]; then \
-		curl -sfL https://install.goreleaser.com/github.com/golangci/golangci-lint.sh | sudo sh -s -- -b $(GOBIN)/ v1.21.0; \
+		curl -sfL https://install.goreleaser.com/github.com/golangci/golangci-lint.sh | sh -s -- -b $(GOBIN)/ v1.21.0; \
```

Comment:
do you still need sudo here before 'sh -s'?

Reply:
If needed, I want user to use `sudo make ...` over prompting for a PW.